server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/LoadFiles.java

```diff
@@ -504,12 +504,15 @@ static long loadFiles(Loader loader, BulkInfo bulkInfo, Path bulkDir,
     }
 
     long sleepTime = loader.finish();
-    if (sleepTime > 0) {
-      log.trace("{}: Tablet Max Sleep is {}", fmtTid, sleepTime);
-      long scanTime = Math.min(totalProcessingTime.toMillis(), 30_000);
-      log.trace("{}: Scan time is {}", fmtTid, scanTime);
-      sleepTime = Math.max(sleepTime, scanTime * 2);
+    if (sleepTime <= 1) {
+      log.trace("{}: Sleeping for {}ms", fmtTid, sleepTime);
+      return sleepTime;
     }
+
+    log.trace("{}: Tablet Max Sleep is {}", fmtTid, sleepTime);
+    long scanTime = Math.min(totalProcessingTime.toMillis(), 30_000);
+    log.trace("{}: Scan time is {}", fmtTid, scanTime);
+    sleepTime = Math.max(sleepTime, scanTime * 2);
```

Comment:
```suggestion
    // sleepTime of 0 or 1 are success cases where we don't want to sleep anymore
    if (sleepTime > 1) {
      log.trace("{}: Tablet Max Sleep is {}", fmtTid, sleepTime);
      long scanTime = Math.min(totalProcessingTime.toMillis(), 30_000);
      log.trace("{}: Scan time is {}", fmtTid, scanTime);
      sleepTime = Math.max(sleepTime, scanTime * 2);
    }
```

Reply:
@ctubbsii  in your suggestions could change the comment to... 1 does not indicate success in that this step is done, it means its probably done but need to run this step again and scan metadata table again to be sure.

```java
 // sleepTime of 0 indicates success.  sleepTime of 1 indicates all rpcs to tservers are done, and we are probably done but still need to run this step agains and scan metadata table one more time to be sure.
```